cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc

```diff
@@ -33,6 +33,14 @@ Full-text schema indexes:
 * can be configured to be _eventually consistent_, in which index updating is moved from the commit path to a background thread.
 Using this feature, it is possible to work around the slow Lucene writes from the performance critical commit process, thus removing the main bottlenecks for Neo4j write performance.
 
+At first sight, the construction of full-text indexes can seem similar to regular indexes.
+However there are some things that are interesting to note:
+In contrast to <<query-schema-index-introduction, regular indexes>>, a full-text index 
+* can be applied to more than one label.
+* can be applied to relationship types (one or more).
+* can be applied to more than one property at a time (similar to a <<schema-index-create-a-composite-index, _composite index_>>) but with an important difference:
+While a composite index applies only to entities that match the indexed label and _all_ of the indexed properties, full-text index will index entities that have at least one of the indexed labels or relationship types, and _any_ of the indexed properties.
```

Comment:
Maybe the last bit would be clearer as `and at least one of the index properties.`

Reply:
Thank you @chrisvest 🙏